spec/forms/schools/on_boarding/admin_contact_spec.rb

```diff
@@ -35,6 +35,16 @@
           expect(subject.errors[:phone]).to be_empty
         end
       end
+
+      context 'correct format with extension' do
+        let :phone do
+          '01234567890 ext 123'
+        end
+
+        it 'is valid' do
```

Comment:
Would `it { is_expected.to allow_value('01234567890 ext 123').for(:phone) }` be a more succinct way of writing this?